editor/db/test/full.pbtxt

```diff
@@ -371,6 +371,11 @@ reactions {
     }
     target_ph: 89.0
     is_automated: true
+    volume {
```

Comment:
The proto doc says these amounts are only for aliquots---should this trigger a validation error?

Reply:
I wasn't sure how strict we wanted to be in workup validations. Right now, we only ensure that if the step is type X, fields {Y} are defined. We don't ensure that irrelevant fields are _not_ defined for any other types

Reply:
OK, maybe something to think about for future PRs, especially if a field is irrelevant for certain types.